gh_pages/lifecycle.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: minimal
+---
+
+## Lifecycle
+
+You can think of respostatus as a way to describe the lifecycle-phase of a project. This may help you to choose an appropriate status for your repositories.
+
+Repostatus aimes to answer two questions about a repository:
+
+* How complete is this? Should I expect it to be "stable" software, mostly working, or not really useful at all?
+* Should I expect active development and support, maybe some bug fixes, or nothing at all?
+
+We wish to talk about whether the content of a repository is in a usable state and how much support from contributors can be expected. Thus statuses can be divided along two metrics: usableness and support.
```

Comment:
I think that "axes" (the plural of axis) would be more idiomatic English. I'd probably also go with "usability".

Reply:
✔️